### PR TITLE
A bunch of small fixes

### PR DESCRIPTION
**Pull request by stv0g**
Hi @konstin,

this are a few commits fixing some broken translations, icons and inconsistencies in the code-style.

### CODE REVIEW

File: importer/import_json.py [hunk 1]
```diff
@@ -41,8 +41,17 @@
         "result",
         "oparl_id",
     ],
-    models.Consultation: ["meeting_id", "paper_id", "oparl_id"],
-    models.File: ["name", "oparl_download_url", "oparl_access_url", "oparl_id"],
+    models.Consultation: [
```
Comment (konstin) [inline importer/import_json.py:44]: meine-stadt-transparent uses [black](https://github.com/psf/black), I've added a ci job in 97fe124b8cfc3fd7a54ad0f46ca3cdf4b239b128 to lint for that

**Comment by konstin** [comment]
thanks for updating the translations and the icons

I think the renamed commands needs to also be changed in the docs, e.g. `citytoags` is still written that way in Import.md

**Comment by stv0g** [comment]
Thanks for the tips,

I fixed the invalid references to the renamed commands and used black for checking the formatting.

The commits are also rebased on the latest main branch, so the new linting CI check is included.

**Comment by konstin** [comment]
Thank you!

**Pull request merged**